src/block/Door.php

```diff
@@ -176,4 +168,29 @@ public function getAffectedBlocks() : array{
 	private function canBeSupportedAt(Block $block) : bool{
 		return $block->getAdjacentSupportType(Facing::DOWN)->hasEdgeSupport();
 	}
+
+	public function onProjectileInteraction(Projectile $projectile) : void{
+		if($projectile instanceof WindCharge) {
+			if($this->getTypeId() === BlockTypeIds::IRON_DOOR) {
+				return;
+			}
+
+			$this->toggle();
```

Comment:
Is the door open when the wind charge interacts with it?

Reply:
it should toggle the door either way it is facing